src/sentry/receivers/releases.py

```diff
@@ -16,23 +16,25 @@ def ensure_release_exists(instance, created, **kwargs):
     if instance.data and instance.data.get('release_id'):
         return
 
+    project = Project.objects.get(pk=instance.project_id)
```

Comment:
What's the point in doing this? Why was wrong with just `instance.project` ? These changes seem entirely unrelated.

Reply:
`instance` is a `TagValue`,

```
project_id = BoundedPositiveIntegerField(db_index=True, null=True)
```

Maybe I'm missing something?

Reply:
Ohh, you're right. I read this a bit too quickly. Thought it was a `Release`. Love types.

Reply:
I actually renamed all the `instance` params because it was annoying me, and then I realized (via tests) that we/Django calls it with kwargs (`instance=foo`). Love Python.

Reply:
![](https://media.giphy.com/media/gVoBC0SuaHStq/giphy.gif)